tests_integration/__tests__/__snapshots__/write.js.snap

```diff
@@ -15,14 +15,14 @@ exports[`do not write file with --write + invalid file (stderr) 1`] = `
 `;
 
 exports[`do not write file with --write + invalid file (stdout) 1`] = `
-"invalid.js
```

Comment:
Not sure if this is supposed to be logged when the file has a syntax error?

Reply:
It should not be logged, but how did it get removed here? I didn't see code changes related to it. 😱

Reply:
Must have just been when I updated the snapshot.